aws/resource_aws_spot_fleet_request.go

```diff
@@ -1118,46 +1118,42 @@ func resourceAwsSpotFleetRequestDelete(d *schema.ResourceData, meta interface{})
 	terminateInstances := d.Get("terminate_instances_with_expiration").(bool)
 
 	log.Printf("[INFO] Cancelling spot fleet request: %s", d.Id())
-	resp, err := conn.CancelSpotFleetRequests(&ec2.CancelSpotFleetRequestsInput{
-		SpotFleetRequestIds: []*string{aws.String(d.Id())},
-		TerminateInstances:  aws.Bool(terminateInstances),
-	})
-
+	err := deleteSpotFleetRequest(d.Id(), terminateInstances, 5*time.Minute, conn)
```

Comment:
Can we upgrade this to support user configurable timeouts, like we do for `CREATE`?

https://github.com/terraform-providers/terraform-provider-aws/blob/68dd50052700ad146d38224ece693471c2ab6045/aws/resource_aws_spot_fleet_request.go#L26-L28